django/contrib/admin/options.py

```diff
@@ -1350,9 +1350,10 @@ def changeform_view(self, request, object_id=None, form_url='', extra_context=No
                     'name': force_text(opts.verbose_name), 'key': escape(object_id)})
 
             if request.method == 'POST' and "_saveasnew" in request.POST:
-                return self.add_view(request, form_url=reverse('admin:%s_%s_add' % (
-                    opts.app_label, opts.model_name),
-                    current_app=self.admin_site.name))
+                # With obj and object_id set to None, the following is executed
```

Comment:
This comment makes sense in light of the change, but might not make sense for someone looking at the code in the future who didn't know that it used `add_view()`